### PR TITLE
V1.41.0 1 stage

### PR DESCRIPTION
**Pull request by mssrivas**
### Summary
Release 1.41.0-1

### Testing

New tests cover the changes: 

### Licensing

This contribution is under the terms of the Apache 2.0 License:

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -29,7 +29,7 @@ static:
 	./scripts/gobuild.sh
 
 govet:
-	go vet $(shell go list ./ecs-init/...)
+	go vet ./...
```
Comment (yunhee-l) [inline Makefile:32]: why was this changed?

File: scripts/changelog/changelog.go [hunk 2]
```diff
@@ -85,7 +85,7 @@ func main() {
 		for scanner.Scan() {
 			line := scanner.Text()
 			if strings.Contains(line, "%changelog") {
-				amazonLinuxSpecBase += fmt.Sprintln("%changelog")
+				amazonLinuxSpecBase += "%changelog"
```
Comment (yunhee-l) [inline scripts/changelog/changelog.go:88]: with removal of Sprintln, should this be "%changelog\n" ?

File: Makefile (hunk 1 above)
Comment (sparrc) [reply Makefile:32]: this was changed because it’s not necessary and it was missing some files in ./scripts/ directory

File: scripts/changelog/changelog.go (hunk 2 above)
Comment (sparrc) [reply scripts/changelog/changelog.go:88]: yes I think you may be correct

Comment (mssrivas) [reply scripts/changelog/changelog.go:88]: i can fix this one
